src/cocaine-app/planner/__init__.py

```diff
@@ -73,12 +78,88 @@ def __init__(self, db, niu, job_processor, namespaces_settings):
                     self._couple_defrag
                 )
 
+            if self.params.get('ttl_cleanup', {}).get('enabled', False):
+                self.__tq.add_task_at(
+                    self.TTL_CLEANUP,
+                    self.ttl_cleanup_timer.next(),
+                    self._ttl_cleanup_planner
+                )
+
     def _start_tq(self):
         self.__tq.start()
 
     def add_planner(self, planner):
         planner.schedule_tasks(self.__tq)
 
+    def _get_yt_stat(self):
+        """
+        Extract statistics from YT logs
+        :param: couples with expired data size > expired_threshold are recommended to be cleaned
+        :return: list of couples with expired records volume above specified
+        """
+        try:
+            # Configure parameters for work with IT
```

Comment:
Done in https://github.com/yandex/mastermind/pull/311/commits/4732106cd00d7b48fb9a6db30b7cff4aef8db6df